.github/workflows/main.yml

```diff
@@ -0,0 +1,272 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "master"
```

Comment:
might be a dumb question, but will this only run on merge rather than individual PRs?

Reply:
This specifies to run the actions on each push against master. The `pull_request` below also adds a trigger for all PRs.

Reply:
dumb question confirmed 🙈